### ISSUE TITLE
Validation Accuracy is 0

### ISSUE THREAD

**Issue by RichardRivaldo** [PaddlePaddle/PaddleOCR#9573]
Here is my SVTR config I modified from #6632 

```
Global:
    use_gpu: False
    epoch_num: 500
    log_smooth_window: 20
    print_batch_step: 10
    save_model_dir: ./output/id/svtr/
    save_epoch_step: 10
    eval_batch_step: [0, 200]
    cal_metric_during_train: True
    pretrained_model:
    checkpoints:
    save_inference_dir:
    use_visualdl: False
    character_dict_path: ppocr/utils/en_dict.txt
    max_text_length: 25
    infer_mode: False
    use_space_char: True

Optimizer:
    name: AdamW
    beta1: 0.9
    beta2: 0.99
    epsilon: 0.00000008
    weight_decay: 0.05
    no_weight_decay_name: norm pos_embed
    one_dim_param_no_weight_decay: true
    lr:
        name: Cosine
        learning_rate: 0.00025
        warmup_epoch: 10

Architecture:
    model_type: rec
    algorithm: SVTR
    Transform:
        name: STN_ON
        tps_inputsize: [32, 64]
        tps_outputsize: [32, 100]
        num_control_points: 20
        tps_margins: [0.05, 0.05]
        stn_activation: none
    Backbone:
        name: SVTRNet
        img_size: [32, 100]
        out_char_num: 25
        out_channels: 192
        patch_merging: "Conv"
        embed_dim: [64, 128, 256]
        depth: [3, 6, 3]
        num_heads: [2, 4, 8]
        mixer:
            [
                "Local",
                "Local",
                "Local",
                "Local",
                "Local",
                "Local",
                "Global",
                "Global",
                "Global",
                "Global",
                "Global",
                "Global",
            ]
        local_mixer: [[7, 11], [7, 11], [7, 11]]
        last_stage: True
        prenorm: false
    Neck:
        name: SequenceEncoder
        encoder_type: reshape
    Head:
        name: CTCHead

Loss:
    name: CTCLoss

PostProcess:
    name: CTCLabelDecode

Metric:
    name: RecMetric
    main_indicator: acc

Train:
    dataset:
        name: SimpleDataSet
        data_dir: ./train_data/rec/train
        label_file_list: ["./train_data/rec/train.txt"]
        transforms:
            - DecodeImage: # load image
                  img_mode: BGR
                  channel_first: False
            - CTCLabelEncode: # Class handling label
            - RecResizeImg:
                  character_dict_path:
                  image_shape: [3, 32, 320]
                  padding: False
            - KeepKeys:
                  keep_keys: ["image", "label", "length"] # dataloader will return list in this order
    loader:
        shuffle: True
        batch_size_per_card: 512
        drop_last: True
        num_workers: 4

Eval:
    dataset:
        name: SimpleDataSet
        data_dir: ./train_data/rec/train
        label_file_list: ["./train_data/rec/val.txt"]
        ratio_list:
            - 0.1
        transforms:
            - DecodeImage: # load image
                  img_mode: BGR
                  channel_first: False
            - CTCLabelEncode: # Class handling label
            - RecResizeImg:
                  character_dict_path:
                  image_shape: [3, 32, 320]
                  padding: False
            - KeepKeys:
                  keep_keys: ["image", "label", "length"] # dataloader will return list in this order
    loader:
        shuffle: False
        drop_last: False
        batch_size_per_card: 256
        num_workers: 2
```

Previously, I used MobileV3 for 500 epochs and my training acc is 0 at the end of the training. After changing the config with the SVTR config, it increased up to 99%. However, the validation accuracy is still 0, just like in this #6803. How do I fix this?

![image](https://user-images.githubusercontent.com/60037073/228430164-1b753435-eaaf-4079-86a0-06aa250552f3.png)

Currently, I use Indonesian image dataset I synthesized of around 5k images. Do I need more data on this? I can provide the data if needed.

**Comment by tink2123** [comment]
How many pictures your evaluation set has, and is it the same distribution as the training data? Try to infer one picture and observe the output results, you should be able to find common problems.

infer one pic:
```
python tools/infer_rec.py -c xxx -o Global.pretrained_model=output/id/svtr/best_accuracy
```

**Comment by RichardRivaldo** [comment]
My evaluation set has approximately 1k images, and yes, it has the same distribution as the training one (3.5k). I did try making an inference but the result is gibberish and very far from the actual label, so the accuracy given is quite understandable. I'm not sure what the problem is. For reference, the image I supplied in the finetuning process are already preprocessed separately and are in black and white. I thought the learning process would actually be easier, but is there any problem regarding this?
